easybuild/easyconfigs/a/AmrPlusPlus/AmrPlusPlus-2.0-GCC-8.3.0.eb

```diff
@@ -0,0 +1,74 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 		https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::		Robert Qiao <rob.qiao@flinders.edu.au>
+# Software License::	MIT
+#
+# Notes::
+##
+
+easyblock = 'CmdCp'
+
+name = 'AmrPlusPlus'
+version = '2.0'
```

Comment:
Since the source you are downloading is not a fixed 2.0 version, I think we should add a datestamp to the version, which corresponds to the date of the commit you're using...

```python
local_commit = '8afc0af5234f8e322cc0f5e40cc553ecd5c932b8'
version = '2.0-20200114'
```